package.json

```diff
@@ -30,6 +30,6 @@
   "devDependencies": {
```

Comment:
To avoid an error, we need to bump the required version in `peerDependencies` as well.

Reply:
Whoops.